lib/src/cli.dart

```diff
@@ -21,7 +21,7 @@ List<String> grinderArgs() => _args;
 List<String> _args;
 bool _verifyProjectRoot;
 
-Future handleArgs(List<String> args, {bool verifyProjectRoot}) {
+Future handleArgs(List<String> args, {bool verifyProjectRoot : false}) {
```

Comment:
I added this default value because a test failed in checked mode because `if(verifyProjectRoot)` with `verifyProjectRoot == null`